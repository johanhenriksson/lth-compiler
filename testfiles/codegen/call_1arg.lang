void f(int a) {
    print(a);
}

void main() {
    f(5);
}
