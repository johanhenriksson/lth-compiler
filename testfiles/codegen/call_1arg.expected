; Library Functions
declare void @print(i64)
declare i64 @read()
declare i64 @trunc(double)
declare double @promote(i64)
declare void @assert(i64)

define void @f(i64 %a) {
call void @print(i64 %a)
ret void
}

define void @main() {
call void @f(i64 5)
ret void
}

